matrix_is_tester/fakehs.py

```diff
@@ -92,19 +104,31 @@ def log_message(self, fmt, *args):
         return
 
 
-def run_http_server():
+def _run_http_server():
     cert_file = os.path.join(os.path.dirname(__file__), "fakehs.pem")
 
-    httpd = BaseHTTPServer.HTTPServer(("localhost", 4490), FakeHomeserverRequestHandler)
+    httpd = BaseHTTPServer.HTTPServer(
+        ("localhost", 4490), _FakeHomeserverRequestHandler
+    )
     httpd.socket = ssl.wrap_socket(httpd.socket, certfile=cert_file, server_side=True)
     httpd.serve_forever()
 
 
 class FakeHomeserver(object):
+    """
+    A class that spawns an HTTP server that looks like a Matrix Homeserver.
+    Currently just implements the federation OpenID endpoint to validate OpenID tokens.
+    """
+
     def launch(self):
-        self.process = Process(target=run_http_server)
+        self.process = Process(target=_run_http_server)
         self.process.start()
 
+        """
```

Comment:
this should be after the `def get_addr` line

Reply:
huh. presumably another bad github merge from when they had issues the other day.